testsuite/unittests/test_precise_neuron_flag.sli

```diff
@@ -0,0 +1,33 @@
+/*
+ *  test_precise_neuron_flag.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+ 
+(unittest) run
+/unittest using
+
+/hh_psc_alpha Create /grid_neuron Set
+grid_neuron GetStatus /is_precise_neuron get false eq assert_or_die
```

Comment:
We generally write code SLI tests as blocks, since we then can see in the output which block failed. Since support for precise spikes is a model property, we can check it on the model defaults (also dropping comparison with true/false):
```
{ /hh_psc_alpha GetDefaults /supports_precise_spikes get not } assert_or_die
{ /parrot_neuron_ps GetDefaults /supports_precise_spikes get } assert_or_die
```